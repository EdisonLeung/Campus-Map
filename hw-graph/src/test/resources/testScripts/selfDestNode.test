# Creates a single node and tests an edge referencing that node (a node that has
# itself as a destination)

# Create graph
CreateGraph selfDest

# Add the node
AddNode selfDest a

# Add the self referencing edge
AddEdge selfDest a a AA

# Print node and children
ListNodes selfDest
ListChildren selfDest a