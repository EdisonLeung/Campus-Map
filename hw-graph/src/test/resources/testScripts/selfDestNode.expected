# Creates a single node and tests an edge referencing that node (a node that has
# itself as a destination)

# Create graph
created graph selfDest

# Add the node
added node a to selfDest

# Add the self referencing edge
added edge AA from a to a in selfDest

# Print node and children
selfDest contains: a
the children of a in selfDest are: a(AA)